src/module-1/cutStrings/index.js

```diff
@@ -1 +1,14 @@
-export const cutStrings = (arr = []) => {};
+export const cutStrings = (arr = []) => {
+  if (arr.length > 0) {
+    let shortestStr = arr.reduce((a, b) => a.length <= b.length ? a : b);
+    let result = [];
+
+    arr.forEach(function (value) {
+      result.push(value.slice(0, shortestStr.length))
+    });
+
+    return result
+  } else {
```

Comment:
you may skip `else` here because the previous statement returns (any code outside that if runs in `else` mode already